UM/Qt/Bindings/Theme.py

```diff
@@ -60,9 +60,9 @@ def fonts(self):
     def sizes(self):
         return self._sizes
 
-    @pyqtSlot(QUrl)
+    @pyqtSlot(str)
```

Comment:
ResourcesProxy now returns a string rather than a URL because I was running into issues with QUrl. This means we should use a string here as well.